From: "Arun Kumar K.R" <akumarkr@codeaurora.org>
Subject: hardware/libhardware/hwcomposer (0001): add sourceTransform to hwc_layer_t

sourceTransform has the source buffer transform which can used to
apply rotation on displays whose orientation is fixed(HDMI).

diff --git a/include/hardware/hwcomposer_v0.h b/include/hardware/hwcomposer_v0.h
index 473819b..6e9308d 100644
--- a/include/hardware/hwcomposer_v0.h
+++ b/include/hardware/hwcomposer_v0.h
@@ -97,6 +97,11 @@ typedef struct hwc_layer {
              * not changed. */
             buffer_handle_t handle;
 
+#ifdef QCOM_HARDWARE
+            /* source transform of the buffer */
+            uint32_t sourceTransform;
+#endif
+
             /* transformation to apply to the buffer during composition */
             uint32_t transform;
 
-- 
1.8.4

